### PR TITLE
[BEAM-1500] Fix typos and bring common_job_properties closer to Groovy style guide.

### PR DESCRIPTION
**Pull request by jasonkuster**
Signed-off-by: Jason Kuster <jasonkuster@google.com>

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by jasonkuster** [comment]
R: @tgroh 

hey buddy

**Comment by kennknowles** [comment]
This style guide? http://groovy-lang.org/style-guide.html

Just curious / want to see if I can get my IDE to autoformat for it, etc.

**Comment by jasonkuster** [comment]
That's the one!

File: .jenkins/common_job_properties.groovy [hunk 1]
```diff
@@ -18,29 +18,31 @@
...
   // Sets common top-level job properties for main repository jobs.
-  static def setTopLevelMainJobProperties(def context,
-                                          def default_branch = 'master') {
-    setTopLevelJobProperties(context, 'beam', default_branch, 100)
+  static void setTopLevelMainJobProperties(context,
+                                           String defaultBranch = 'master') {
+    setTopLevelJobProperties(context, 'beam', defaultBranch, 100)
   }
 
   // Sets common top-level job properties. Should be accessed through one of the
```
Comment (tgroh) [inline .jenkins/common_job_properties.groovy]: should or must?

File: .jenkins/common_job_properties.groovy [hunk 2]
```diff
@@ -18,29 +18,31 @@
...
-                                              def default_timeout) {
+  private static void setTopLevelJobProperties(context,
+                                               String repositoryName,
+                                               String defaultBranch,
+                                               String defaultTimeout) {
 
     // GitHub project.
     context.properties {
-      githubProjectUrl('https://github.com/apache/' + repository_name + '/')
+      githubProjectUrl('https://github.com/apache/' + repositoryName + '/')
```
Comment (tgroh) [inline .jenkins/common_job_properties.groovy]: `"https://github.com/apache/${repositoryName}/"` or `"https://github.com/apache/$repositoryName/"`

File: .jenkins/common_job_properties.groovy [hunk 3]
```diff
@@ -59,7 +61,7 @@ class common_job_properties {
     context.scm {
       git {
         remote {
-          url('https://github.com/apache/' + repository_name + '.git')
+          url('https://github.com/apache/' + repositoryName + '.git')
```
Comment (tgroh) [inline .jenkins/common_job_properties.groovy]: Same string interpolation

File: .jenkins/common_job_properties.groovy [hunk 4]
```diff
@@ -98,24 +100,24 @@ class common_job_properties {
...
         permitAll()
-        // trigger_phrase is the argument which gets set when we want to allow
+        // prTriggerPhrase is the argument which gets set when we want to allow
         // post-commit builds to run against pending pull requests. This block
         // overrides the default trigger phrase with the new one. Setting this
         // will disable automatic invocation of this build; the phrase will be
         // required to start it.
-        if (trigger_phrase != '') {
-          triggerPhrase(trigger_phrase)
+        if (prTriggerPhrase != '') {
```
Comment (tgroh) [inline .jenkins/common_job_properties.groovy]: `if (prTriggerPhrase)`

empty is falsy

File: .jenkins/common_job_properties.groovy (hunk 1 above)
Comment (jasonkuster) [inline .jenkins/common_job_properties.groovy]: Done

File: .jenkins/common_job_properties.groovy (hunk 2 above)
Comment (jasonkuster) [inline .jenkins/common_job_properties.groovy]: Done

File: .jenkins/common_job_properties.groovy (hunk 3 above)
Comment (jasonkuster) [inline .jenkins/common_job_properties.groovy]: Done

File: .jenkins/common_job_properties.groovy (hunk 4 above)
Comment (jasonkuster) [inline .jenkins/common_job_properties.groovy]: Done

**Comment by jasonkuster** [comment]
@tgroh PTAL

File: .jenkins/common_job_properties.groovy [hunk 5]
```diff
@@ -18,29 +18,31 @@
 
 // Contains functions that help build Jenkins projects. Functions typically set
 // common properties that are shared among all Jenkins projects.
+// Code in this directory should conform to the Groovy style guide.
```
Comment (tgroh) [inline .jenkins/common_job_properties.groovy]: This seems like not a file comment.

**Comment by jasonkuster** [comment]
Run Seed Job

**Comment by jasonkuster** [comment]
Run Seed Job

**Comment by jasonkuster** [comment]
@tgroh - had to revert string changes, somehow it looks like Jenkins parsing of the groovy doesn't support them. Tested seed job and it succeeded and all configs look sane; this is ready for merge.

**Comment by tgroh** [comment]
That's unfortunate. Oh well.
LGTM

**Pull request closed without merging**